### PR TITLE
Add action/button to reset user(s) session(s) in user admin

### LINKED ISSUE #13705: Allow admins to invalidate a user session
Admins need a way to invalidate a user session to force a re-login into AMO.

We would probably want to add a button or action to the django admin user profile page that would clear `auth_id` for that user.

### PR DESCRIPTION
**Pull request by diox**
Fixes #13705

### CODE REVIEW

File: src/olympia/users/admin.py [hunk 1]
```diff
@@ -169,7 +173,24 @@ def reset_api_key_view(self, request, object_id, extra_context=None):
         if not acl.action_allowed(request, amo.permissions.USERS_EDIT):
             return HttpResponseForbidden()
 
-        self.reset_api_key_action(request, [obj])
+        self.reset_api_key_action(
+            request, UserProfile.objects.filter(pk=obj.pk))
+
+        return HttpResponseRedirect('../')
```
Comment (willdurand) [inline src/olympia/users/admin.py]: ha?

Comment (willdurand) [reply src/olympia/users/admin.py]: I see this used already but returning the full URL seems a bit more future-proof to me.

File: src/olympia/users/admin.py [hunk 2]
```diff
@@ -169,7 +173,24 @@ def reset_api_key_view(self, request, object_id, extra_context=None):
...
+        if obj is None:
+            raise Http404()
+
+        if not acl.action_allowed(request, amo.permissions.USERS_EDIT):
+            return HttpResponseForbidden()
+
+        self.reset_session_action(
+            request, UserProfile.objects.filter(pk=obj.pk))
 
         return HttpResponseRedirect('../')
```
Comment (willdurand) [inline src/olympia/users/admin.py]: Same here although this was clearly there before.

File: src/olympia/users/templates/admin/users/userprofile/change_form.html [hunk 3]
```diff
@@ -15,6 +15,8 @@
                          type="submit">{% trans "Ban" %}</button>
                  <button formaction="{% url 'admin:users_userprofile_reset_api_key' original.pk|admin_urlquote %}"
                          type="submit">{% trans "Reset API Key" %}</button>
+                 <button formaction="{% url 'admin:users_userprofile_reset_session' original.pk|admin_urlquote %}"
```
Comment (willdurand) [inline src/olympia/users/templates/admin/users/userprofile/change_form.html]: We should have some style for these buttons, they look quite ugly compared to the others:

Screen Shot 2020-03-19 at 09 54 36

Comment (diox) [reply src/olympia/users/templates/admin/users/userprofile/change_form.html]: https://github.com/mozilla/addons-server/pull/13763/commits/2c6185cdf88016fd81890c7dfa84092c727597e9

File: src/olympia/users/admin.py (hunk 2 above)
Comment (diox) [reply src/olympia/users/admin.py]: https://github.com/mozilla/addons-server/pull/13763/commits/20814c702404863bbb56f2e2b73d8d609b1791f9

File: src/olympia/users/admin.py (hunk 1 above)
Comment (diox) [reply src/olympia/users/admin.py]: https://github.com/mozilla/addons-server/pull/13763/commits/20814c702404863bbb56f2e2b73d8d609b1791f9

**Pull request merged**